runbotics/runbotics-scheduler/src/database/process/process.service.ts

```diff
@@ -30,4 +32,34 @@ export class ProcessService {
     partialUpdate(process: PartialUpdateProcess) {
         return this.processRepository.update({ id: process.id }, process);
     }
+
+    async checkAccessAndGetById(processId: number, user: UserEntity) {
+        // TODO: to be updated after process migration to scheduler
```

Comment:
we've settled to allow for running the process with credentials if user does have access to the process, so this todo is not necessary after all. @socjalis correct me if I am mistaken. 

Sorry for not pointing this out in FigJam, I'll do it

Reply:
"if user does have access to the process" - and this function is actually checking this, but how u can see it represents the old approach without process collections check. To check privilege to process we need to check privilege to its collection and all above.

In case of UI probably it doesn't have to be checked, but in API definitely imo.